examples/add_public_raster_dataset.ipynb

```diff
@@ -0,0 +1,304 @@
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[Volume(id=f6aa9f3d-a211-43e8-9b91-b23ab9632791, path='test_data/')]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "volumes = ge.volumes()\n",
+    "volumes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define metadata"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'type': 'GdalStatic',\n",
+       " 'time': None,\n",
+       " 'params': {'filePath': 'raster/landcover.tif',\n",
+       "  'rasterbandChannel': 1,\n",
+       "  'geoTransform': {'originCoordinate': {'x': -180.0, 'y': 90.0},\n",
+       "   'xPixelSize': 0.1,\n",
+       "   'yPixelSize': -0.1},\n",
+       "  'width': 3600,\n",
+       "  'height': 1800,\n",
+       "  'fileNotFoundHandling': 'NoData',\n",
+       "  'noDataValue': None,\n",
+       "  'propertiesMapping': None,\n",
+       "  'gdalOpenOptions': None,\n",
+       "  'gdalConfigOptions': None,\n",
+       "  'allowAlphabandAsMask': True},\n",
+       " 'resultDescriptor': {'type': 'raster',\n",
+       "  'dataType': 'U8',\n",
+       "  'measurement': {'type': 'classification',\n",
+       "   'measurement': 'Land Cover',\n",
+       "   'classes': {0: 'Water Bodies',\n",
+       "    1: 'Evergreen Needleleaf Forests',\n",
+       "    2: 'Evergreen Broadleaf Forests',\n",
+       "    3: 'Deciduous Needleleaf Forests',\n",
+       "    4: 'Deciduous Broadleleaf Forests',\n",
+       "    5: 'Mixed Forests',\n",
+       "    6: 'Closed Shrublands',\n",
+       "    7: 'Open Shrublands',\n",
+       "    8: 'Woody Savannas',\n",
+       "    9: 'Savannas',\n",
+       "    10: 'Grasslands',\n",
+       "    11: 'Permanent Wtlands',\n",
+       "    12: 'Croplands',\n",
+       "    13: 'Urban and Built-Up',\n",
+       "    14: 'Cropland-Natural Vegetation Mosaics',\n",
+       "    15: 'Snow and Ice',\n",
+       "    16: 'Barren or Sparsely Vegetated'}},\n",
+       "  'spatialReference': 'EPSG:4326'}}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "file_path = \"raster/landcover.tif\" # specify file path relative to volume path\n",
+    "\n",
+    "rasterband_channel = 1\n",
+    "\n",
+    "origin = [-180.0, 90.0]\n",
+    "x_pixel_size = 0.1\n",
+    "y_pixel_size = -0.1\n",
+    "geo_transform = ge.GdalDatasetGeoTransform(origin, x_pixel_size, y_pixel_size)\n",
+    "\n",
+    "width = 3600\n",
+    "height = 1800\n",
+    "\n",
+    "file_not_found_handling = ge.FileNotFoundHandling.NODATA\n",
+    "\n",
+    "gdal_params = ge.GdalDatasetParameters(file_path, rasterband_channel, geo_transform, width, height, file_not_found_handling, None, None, None, None, True)\n",
```

Comment:
Maybe you can use the attribute writing, because you don't know what these Nones mean.

If you have a type with too many parameters, you should either change it, or at least provide the example where you just name every attribute.

file_path = 'foo', rasterband_channel=2, geo_transform=…

Reply:
> If you have a type with too many parameters, you should either change it,

this would mean changing the API

> or at least provide the example where you just name every attribute.
file_path = 'foo', rasterband_channel=2, geo_transform=…

well, I already named the variables accordingly. Shall I really write `(file_path = file_path, rasterband_channel = rasterband_channel`?